magento/ece-tools
# Adds Elasticsearch 7.5 to ECE tools for Magento 2.3.5

### Description
Adds Elasticsearch 7.5 to ECE tools for Magento 2.3.5

This is related to:  https://github.com/magento/magento-cloud-docker/pull/143

### Fixed Issues (if relevant)
1. magento/magento-cloud-docker#141 Add Elasticsearch 7.5 support
2. magento/ece-tools#681 Add Elasticsearch 7.5 support


### Manual testing scenarios
1.  Test with 2.3.5
2. Set up docker with Elasticsearch 7.5
3. Ensure that the ece-tools does not throw warnings about elasticsearch version

### Contribution checklist
